internal/cloud/e2e/migrate_state_tfc_to_tfc_test.go

```diff
@@ -348,8 +348,8 @@ func Test_migrate_tfc_to_tfc_multiple_workspace(t *testing.T) {
 				},
 			},
 			validations: func(t *testing.T, orgName string) {
-				wsList, err := tfeClient.Workspaces.List(ctx, orgName, tfe.WorkspaceListOptions{
-					Tags: tfe.String("billing"),
+				wsList, err := tfeClient.Workspaces.List(ctx, orgName, &tfe.WorkspaceListOptions{
+					Tags: "billing",
```

Comment:
Same question here - would we want to do something like `[]tfe.WSIncludeOpt{tfe.WSCurrentRun}}` here?